docs/backups.md

```diff
@@ -1,25 +1,67 @@
+* [Azure Blob Storage :octicons-link-external-16:](https://azure.microsoft.com/en-us/services/storage/blobs/):
 
 ![image](assets/images/backup-cloud.svg)
 
-But storing backups on [Persistent Volumes :octicons-link-external-16:](https://kubernetes.io/docs/concepts/storage/persistent-volumes/) inside the Kubernetes cluster is also possible:
+If you're running a Kubernetes cluster on premises, you can  store backups inside it using a [Persistent Volume :octicons-link-external-16:](https://kubernetes.io/docs/concepts/storage/persistent-volumes/). For example, if you don't use a remote backup storage or if storage costs are high for you. 
 
 ![image](assets/images/backup-pv.svg)
 
-The Operator does logical backups, querying Percona XtraDB Cluster for the
-database data and writing the retrieved data to the backup storage. The backups
-are done using the [Percona XtraBackup :octicons-link-external-16:](https://docs.percona.com/percona-xtrabackup/2.4/index.html) tool.
+## Workflow
+
+After you create a Backup object, the Operator sets up a backup Pod that runs Percona XtraBackup inside. It also creates a path in the storage to save the backup data.
+
+The backup Pod starts copying the data files from the Percona XtraDB Cluster  to the backup storage. The Percona XtraDB Cluster Pod that serves the data enters the Donor state and stops receiving all requests.
+
+The backup task is resource-consuming and can affect performance. That's why the Operator uses one of the secondary Percona XtraDB Cluster Pods for backups. The exception is a one-pod deployment, where the same Pod is used for all tasks.
+
+After the data files are copied, the Operator marks the backup Pod as 'Completed' and deletes it. The Operator also updates the status of the Backup object.
+
+
+## Multiple backups
+
+You can run several backups. For example, schedule weekly backups on one storage and daily backups on another one. You can also run an on-demand backup to be on the safe side before you do some maintenance work.  
+
+Several backups run in parallel by default if they happen at the same time. If they overload your cluster, you can turn off parallel backups with the `backup.allowParallel` configuration option in the `cr.yaml` file. Then, the Operator queues the backups and runs them sequentially. 
+
+The Operator ensures the sequence by creating a lock for a running backup. It releases the lock after the backup either succeeds or fails and starts the next one from the queue. The lock is also released if you delete a running backup. 
+
+You can fine-tune the queue by assigning a waiting time for a backup to start. Use the `spec.startingDeadlineSeconds` option in the `deploy/cr.yaml` file to set this time for all backups. You can also override it for a specific backup by defining the `startingDeadlineSeconds` option within the backup configuration. This setting has a higher priority.
```

Comment:
Maybe better: You can also override it for a specific `on-demand` backup? We can not override it for scheduled ones.

Reply:
Fixed